docs/examples.rst

```diff
@@ -2,18 +2,39 @@ Example Deployments
 ===================
 
 Deploying dask-jobqueue on different clusters requires a bit of customization.
-Below, we provide a few example deployments:
+Below, we provide a few examples from real deployments in the wild:
 
 
-Example PBS Deployment
-----------------------
+PBS Deployments
+---------------
 
 .. code-block:: python
 
    from dask_jobqueue import PBSCluster
 
-   cluster = PBSCluster(processes=6, threads=4, memory="16GB")
-   cluster.start_workers(10)
+   cluster = PBSCluster(queue='regular',
+                        project='DaskOnPBS',
+                        local_directory=os.getenv('TMPDIR', '/tmp'),
+                        threads=4,
+                        processes=6,
+                        memory='16GB',
+                        resource_spec='select=1:ncpus=24:mem=100GB')
```

Comment:
should we remove this example? I don't think there is anything here (except local_directory) that isn't in the example below.

Reply:
Good to know. I'm fine leaving both or adding an example from your cluster. The `DaskOnPbs` account number is what lead me to think this was not a complete example.

Reply:
Project given through #PBS -A in our HPC is currently purely informal. We can put anything we want in there, even nothing :slightly_smiling_face:. So yes, this example may come from me!